app/next-client-app/app/layout.tsx

```diff
@@ -1,18 +1,7 @@
 import "./globals.css";
-import type { Metadata } from "next";
 import { Toaster } from "sonner";
 import { ThemeProvider } from "@/components/theme-provider";
 
-export const metadata: Metadata = {
```

Comment:
By moving this to the public layout, there is no metadata in the protected pages though (which is still useful to have). 
Why not put the metadata from public in this root layout?

Reply:
My intention was to let the users only find the landing page about Carrot on Google, not the deeper or protected pages. But maybe they also want to see if the protected pages exist as well?

Reply:
Yes this metadata is not just used by search engines, but also browsers (title, manifest) and link sharing. So it's nice to have on all pages.